<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/OpenInfraPlatform.qrc">
    <normaloff>:/icons/icon.ico</normaloff>:/icons/icon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidgetCommon">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabClearColor">
      <attribute name="title">
       <string>Viewport</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>511</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="topMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Clear Color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxGradientClear">
          <property name="text">
           <string>Gradient clear</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxShowGrid">
          <property name="text">
           <string>Show Grid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxShowReferenceCoordinateSystem">
          <property name="text">
           <string>Show Reference Coordinate System</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxSkybox">
          <property name="text">
           <string>Skybox</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxShowViewCube">
          <property name="text">
           <string>Show View Cube</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxShowFrameTimes">
          <property name="text">
           <string>Show Frame Times</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Alignment line width</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBoxAlignmentLineWidth">
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="value">
             <double>1.500000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabCamera">
      <attribute name="title">
       <string>Camera</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>511</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="topMargin">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Smooth View Transition Time</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBoxAnimationTime">
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="value">
             <double>1.170000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Near Clipping Plane Distance</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBoxNearPlane">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Far Clipping Plane Distance</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBoxFarPlane">
            <property name="maximum">
             <double>999999.989999999990687</double>
            </property>
            <property name="value">
             <double>500.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Spectator Camera Mode Only</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Pan Speed</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabRenderSystem">
      <attribute name="title">
       <string>Render System</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Render System</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxRenderSystem">
           <property name="currentText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxMSAA">
         <property name="text">
          <string>Enable Multisample anti-aliasing</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Common</string>
      </attribute>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Language</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxLanguage">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources/OpenInfraPlatform.qrc"/>
 </resources>
 <connections/>
</ui>
